README.md

```diff
@@ -1,6 +1,13 @@
 <div align="center">
   <img src="resources/lmdeploy-logo.png" width="450"/>
 
+[![docs](https://img.shields.io/badge/docs-latest-blue)](https://lmdeploy.readthedocs.io/en/latest/)
+[![badge](https://github.com/InternLM/lmdeploy/workflows/lint/badge.svg)](https://github.com/InternLM/lmdeploy/actions)
+[![codecov](https://codecov.io/gh/InternLM/lmdeploy/branch/main/graph/badge.svg)](https://codecov.io/gh/InternLM/lmdeploy)
```

Comment:
codecov shows unknown. May remove it. Can we add a version badge?

Reply:
yes